machine-learning/tutorials/SentimentAnalysis/Program.cs

```diff
@@ -119,20 +119,21 @@ public static void Evaluate(MLContext mlContext, ITransformer model, IDataView s
             // The Accuracy metric gets the accuracy of a model, which is the proportion 
             // of correct predictions in the test set.
 
-            // The AreaUnderRocCurve metric is an indicator of how confident the model is 
-            // correctly classifying the positive and negative classes as such.
+            // The AreaUnderROCCurve metric is equal to the probability that the algorithm ranks
+            // a randomly chosen positive instance higher than a randomly chosen negative one
+            // (assuming 'positive' ranks higher than 'negative').
 
             // The F1Score metric gets the model's F1 score.
-            //  F1 is a measure of tradeoff between precision and recall.
+            // The F1 score is the harmonic mean of precision and recall:
```

Comment:
I guess if we say mean, it is implied that it mixes both precision and recall into one metric.